gateway/src/main/java/io/zeebe/gateway/grpc/GrpcErrorMapper.java

```diff
@@ -113,22 +105,20 @@ private Status mapBrokerErrorToStatus(final BrokerError error) {
         break;
       case RESOURCE_EXHAUSTED:
         builder.setCode(Code.RESOURCE_EXHAUSTED_VALUE);
+        logger.trace("Target broker is currently overloaded: {}", error);
         break;
       case PARTITION_LEADER_MISMATCH:
         // return UNAVAILABLE to indicate to the user that retrying might solve the issue, as this
         // is usually a transient issue
+        logger.trace("Target broker was not the leader of the partition: {}", error);
```

Comment:
One thing: I thought about setting it to debug, since it may be useful sometimes to know if we have leader mismatch - however you can see that via metrics, and here, well, we don't have request context info so I'm not sure it gives any more insight.